web/pandas/about/sponsors.md

```diff
@@ -11,31 +11,50 @@ health and sustainability of the project. Visit numfocus.org for more informatio
 Donations to _pandas_ are managed by NumFOCUS. For donors in the United States, your gift is tax-deductible
 to the extent provided by law. As with any donation, you should consult with your tax adviser about your particular tax situation.
 
-## Tidelift
+## Become a sponsor
 
-_pandas_ is part of the [Tidelift subscription](https://tidelift.com/subscription/pkg/pypi-pandas?utm_source=pypi-pandas&utm_medium=referral&utm_campaign=readme).
-You can support pandas by becoming a Tidelift subscriber.
+As a free and open source project, _pandas_ relies on the support of the community of users for its development.
+If you work for an organization that uses and benefits from _pandas_, please consider supporting pandas. There
+are different ways, such as employing people to work on pandas, funds the project, or becoming a
```

Comment:
"funding" instead of "funds", to be consistent with "employing" and "becoming"?

Reply:
Yep, thanks!